test/distributed/test_c10d_common.py

```diff
@@ -1378,11 +1385,20 @@ class CommResult:
     _work: torch.classes.c10d.Work
 
 
-def wrap_comm_result(result):
+def wrap_comm_result(result: Tuple[Any]) -> Tuple[Any]:
+    def wrap(work, e):
+        assert isinstance(e, torch.Tensor), (
+            "Excepting collection of tensors as the first element in the "
```

Comment:
“expecting"?

Reply:
Let me update on top of the stack to avoid triggering CI again.